pyflakes/checker.py

```diff
@@ -641,6 +642,15 @@ def descendantOf(self, node, ancestors, stop):
                 return True
         return False
 
+    def getAncestor(self, node, ancestor_type):
+        parent = node
+        while True:
+            if parent is self.root:
```

Comment:
do you really need this?  Please check.

And due to subtree support, I think it is best to prefix this function name with `_` as the subtree support may need to be slightly adjusted as we learn more.